velox/type/HugeInt.h

```diff
@@ -56,6 +56,13 @@ class HugeInt {
   }
 
   static int128_t parse(const std::string& str);
+
+  /// Because value exceeds maxium int64_t cannot write, add this helper
+  /// function to test. It can represent -9223372036854775817223372036854775808
+  /// to 9223372036854775816223372036854775807.
+  /// Example Usage:
+  /// combine(1, 13579) return 1'000'000'000'000'013'579
+  static int128_t combine(int64_t hi, int64_t lo);
```

Comment:
The API looks similar with the build function. Can we clarify their differences in the comments?
https://github.com/facebookincubator/velox/blob/9708e57cb26c40f27bf6f3451b1852a9ff66f912/velox/type/HugeInt.h#L33-L36